js/Heap.js

```diff
@@ -62,15 +62,42 @@ export default Package.create({
           componentThis.state.touchable &&
           componentThis.state.touchable.touchState;
 
+        const targetText = getTargetText(componentThis._reactInternalFiber);
+
         track(eventType, {
           touchableHierarchy,
           touchState,
+          targetText,
         });
       },
     };
   },
 });
 
+// :TODO: (jmtaber129): Consider implementing sibling target text.
+const getTargetText = fiberNode => {
```

Comment:
If the target text is empty, we're returning `''` which will be a little strange downstream in our UI. I propose we instead set it to null and filter out the prop from the blob entirely, in the `track` call. Wdyt?

Reply:
Oh, I see – you're using the empty but non-null string for purposes of recursion. We could do something outside of here in the caller to not send `''` to the server. Thoughts?

Reply:
Yea, when there's no target text on the web, that prop is just not sent.  Updated to only include that prop when it's non-empty